model/general.go

```diff
@@ -15,3 +15,11 @@ const (
 	// NonApplicableState represents constant use for StateChangeEvents when resource did not have a previous state.
 	NonApplicableState = "n/a"
 )
+
+type LaunchTemplateData struct {
```

Comment:
Actually I was looking for a file `ec2.go` and also I didn't see the `aws_types.go`. So instead of creating a new file, I put it here. Now I will place this in `aws_types.go`.